### ISSUE TITLE
[BUG] cannot get controllable with MyWoosh

### ISSUE THREAD

**Issue by Jamesie69** [cagnulein/qdomyos-zwift#2446]
I have switched to myWhoosh but cannot get my Proform TDF5 to be compatible with the Controllable options for inclination.

I can get MyWhoosh to connect for Power but nothing else.
If I use power from Speedplay PWR pedals MyWoosh can see QZ as secondary power source but not cadence or HRT or controllable.

Running QZ on OnePlus Android 14 phone linked to OnePlus Tab on Android 14.

Had no issues with controllable on Zwift or Rouvy.

Hope you can help
James

[debug-Thu_Jul_11_16_29_04_2024.log](https://github.com/user-attachments/files/16180328/debug-Thu_Jul_11_16_29_04_2024.log)

[debug-Thu_Jul_11_16_52_37_2024.log](https://github.com/user-attachments/files/16180330/debug-Thu_Jul_11_16_52_37_2024.log)

**Comment by cagnulein** [comment]
everything on the QZ side is fine. Did you try to rename your android device with QZ to "bike" and reboot it?

**Comment by Jamesie69** [comment]
Hi Roberto,

I renamed my phone to 'bike' as requested and rebooted. I then paired my phone to my tablet.

In MyWhoosh I got a different outcome when connecting to QZ, I had to repair and once I selected the Power source ie, bike, myWhoosh tried to connect for cadence and controllable source, see image attached. 

I tried a myWhoosh ride but it did not move indication that no data is being sourced from my phone. The data fields on my phone indicate QZ is receiving from the TDF 5.

Any thoughts on this?
![IMG20240711182137](https://github.com/cagnulein/qdomyos-zwift/assets/175335736/869b9089-9f08-4644-9a84-387d96c9dbc3)

**Comment by cagnulein** [comment]
Try to swap the devices, use qz on the tablet and mywhoosh on the phone. It seems that there is a cache issue. Maybe you can try also to reinstall mywhoosh

**Comment by Jamesie69** [comment]
I think I solved it.....

I deleted data and cache from the phone, uninstalled QZ, rebooted and reinstalled the QZ app.

On myWhoosh I repaired and got a positive connection on the bike with and inclination reset!

I'll have a go and let you know but all looks positive. Many thanks for your help. Think I owe you a coffee!

Best
James
![IMG20240711184502](https://github.com/cagnulein/qdomyos-zwift/assets/175335736/115e3227-76f4-4612-bada-1346e4dfeb7d)
